### DISCUSSION TITLE
用dperf 压测dpvs服务，在客户端配置好cps，再配置cc 其结果是一样的，配置的cc这个并发参数没用，想了解下这个是什么原因？

### DISCUSSION THREAD

**Discussion by fishyu-mushroom** [baidu/dperf#417] [Q&A]
client 配置一：
mode            client
cpu               0 1 2 3 4 5 6 7
slow_start     60
tx_burst        128
launch_num 10
payload_size 1
duration        90s
protocol        tcp
cps                1290000
port               0000:82:00.0    10.26.3.120    10.26.3.254
client             10.26.3.120    50
server            10.26.3.12      8
listen             80                 1
结果：
![企业微信截图_1708595962510](https://github.com/baidu/dperf/assets/46886228/c7382261-e12f-417c-a09f-15164360fcc2)

client 配置二：
mode            client
cpu               0 1 2 3 4 5 6 7
slow_start     60
tx_burst        128
launch_num 10
payload_size 1
duration        90s
protocol        tcp
cps                1290000
cc                 10000
keepalive      2ms
port               0000:82:00.0    10.26.3.120    10.26.3.254
client             10.26.3.120    50
server            10.26.3.12      8
listen             80                 1
结果：
![企业微信截图_17085961251151](https://github.com/baidu/dperf/assets/46886228/354ab38c-5ec3-4bf1-91de-c21b5b24917c)

**Comment by pengjianzhang** [comment]
服务器端要开启长连接
如果喜欢dperf 请加个star 谢谢
